packages/main/src/Icon.js

```diff
@@ -98,6 +123,16 @@ class Icon extends UI5Element {
 		return icon.d;
 	}
 
+	get hasIconTooltip() {
+		return this.showTooltip && this.accessibleNameText;
+	}
+
+	get accessibleNameText() {
+		const icon = getIconData(this._normalizeIconURI(this.src));
+
+		return this.accessibleName || icon.accText;
```

Comment:
getResourceBundle().getText() should be here

Reply:
We have to fix it first to not break. It should return the english text if no text is found in the translation

Reply:
done